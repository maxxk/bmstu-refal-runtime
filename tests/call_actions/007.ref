$ENTRY Go
{
	= <Prout <Func1 'BeginMiddleEnd'>><Prout <Func2 'BeginMiddleEnd'>>
}

Func1
{
	e.1 e.2 e.3 e.4, e.1 -> ?{'Begin' = Success1} : s.1, e.2 -> ?{'Middle' = Success2} : s.2, e.3 -> ?{'End' = Success3} : s.3 = s.1 s.2 s.3;
}

Func2
{
	e.1 e.2 e.3 e.4, e.1 -> ?{'Begin' = Success1} : s.1, e.2 -> ?{'Middle' = Success2} : s.2, e.3 => ?{'End' = Success3} : s.3 = s.1 s.2 s.3;
}


