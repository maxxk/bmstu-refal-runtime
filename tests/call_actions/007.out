Success1 Success2 Success3 
[VM]: Func call failed!
